client/submitAssignment.js

```diff
@@ -120,7 +143,13 @@ function resetValidations() {
     formError.innerHTML = '';
 }
 
-// Verifies the file type (should be .zip, .tar or .tar.gz)
+/**
+ * Verifies the selected file input for valid file types and extensions.
+ * Allowed file types are 'application/zip', 'application/x-tar','application/gzip'.
```

Comment:
As allowed file types and extensions can change, having this in the comments would mean that we have to keep this in sync with updates to the constants. Instead, we can just mention "Verifies the selected file input for valid file types and extensions."

Reply:
Okay